build.gradle

```diff
@@ -28,6 +28,9 @@ allprojects {
     }
 }
 
+targetCompatibility = 10
```

Comment:
If you move this up to the `allprojects` section of this file (line 18), then you won't need to have it in each subproject's build.gradle (i.e. mod*/build.gradle) -- it will apply it to each one

Reply:
good suggestion! will change it.